docs/README.md

```diff
@@ -613,6 +614,33 @@ Send this query to `/v2/data/tezos/<network>/accounts`
 }
 ```
 
+#### Blocks by end-user-transactions outside April 2019
+
+Send this query to `/v2/data/tezos/<network>/operations`
+
+```json
+{
+    "fields": ["block_level", "operation_group_hash"],
+    "predicates": [
+        { "field": "timestamp", "set": [1554076800000], "operation": "before", "inverse": false, "group": "before" },
+        { "field": "kind", "set": ["transaction"], "operation": "in", "inverse": false, "group": "before" },
```

Comment:
Will group names other "before"/"after" be considered? Or more than two groups, or is this intentionally to restrict "or" nesting?

Reply:
group names are arbitrary. there is no nesting

Reply:
Do you think I might be making a more detailed explanation of how the example is interpreted?

Reply:
It's probably fine, I just got confused because "before" and "after" is a binary way at looking at groups. Maybe different group names would be enough.